api/models/languages_levels.py

```diff
@@ -1,21 +1,25 @@
 from django.db import models
 
-from api.models.base import InternalModelWithName
-
 # Sometimes the languages and levels are needed separately, sometimes in combinations.
 # This means that we cannot create a "choices" field for language or level.
 
 
-class Language(InternalModelWithName):
+class Language(models.Model):
```

Comment:
Why you decided to get rid of InternalModelWithName? 

I agree with this solution, just interesting

Reply:
see commit message in afb89c99a314dc3777a5ae15c3467fcbd5712b58 :)